src/chat/chatter.py

```diff
@@ -248,19 +250,16 @@ def updateUser(self):
         self._verifySortOrder()
 
     def updatePlayer(self):
-        self.set_color()
-
         player = self.user_player
         # Weed out IRC users and those we don't know about early.
         if player is None:
             self.rankItem.setIcon(util.THEME.icon("chat/rank/civilian.png"))
             self.rankItem.setToolTip("IRC User")
             return
 
-        country = player.country
-        if country is not None:
-            self.setIcon(util.THEME.icon("chat/countries/%s.png" % country.lower()))
-            self.setToolTip(country)
+        country = player.country  # server sends '' for no ip2country-resolution
+        self.setIcon(util.THEME.icon("chat/countries/%s.png" % (country or '__').lower()))
```

Comment:
Changed logic, now we're checking truthiness of 'country' instead of comparing it to None. If it can be an empty string, it'd be better to check explicitly.

Reply:
The None case does not happen here, but the '' does.
I thought you like it compact. You want some if ... else ... ?

Reply:
We can do something like `country = player.country if player.country != '' else '__'`. It's not perfect, but Python doesn't have a Maybe monad just yet :/

Reply:
did that